dags/examples/xpk_example_dag.py

```diff
@@ -49,7 +49,7 @@
       tpu_cores=128,
       tpu_zone=Zone.US_CENTRAL2_B.value,
       test_name="resnet-multi-slice",
-      project_name=Project.TPU_PROD_ENV_MULTIPOD.value,
+      project_name=Project.CLOUD_TPU_MULTIPOD_DEV.value,
```

Comment:
It seems like we could make cluster changes like this much simpler by keeping all the cluster details together, like cluster name, device type, zone, etc. @RissyRan is there still a refactor planned for the codebase? Wondering if that's the right place for such improvements or if this would need to be its own work item.

Reply:
+1

Reply:
SGTM! Please feel free to refactor it.